src/tests/remote.test.ts

```diff
@@ -64,4 +81,17 @@ describe('Perl debugger connection', () => {
 		assert.equal(res.ln, 7); // The first code line in test.pl is 7
 	});
 
+	it.skip('Should be able to get loaded scripts and their source code from' + FILE_TEST_PL, async () => {
+
+		const [ server, local ] = setupDebugger(
+			conn, FILE_TEST_PL, DATA_ROOT, [], launchOptions
+		);
+
+		// TODO(bh): Test for `loadedSourcesRequest` and `sourceRequest`.
+		// Those would need a `DebugClient` and/or a `DebugSession`. Not
+		// Clear how to get those here, or how to restructure the tests.
```

Comment:
We could go down a level and test this in the `connection.test.ts` - testing `getSourceCode?`